apps/ledger-live-mobile/src/screens/SyncOnboarding/LanguageSelect.tsx

```diff
@@ -65,24 +65,25 @@ const LanguageSelect = () => {
   // Handles a newly selected language to redux-dispatch
   useEffect(() => {
     if (selectedLanguage) {
-      const newDirection = i18next.dir(selectedLanguage);
-      const currentDirection = I18nManager.isRTL ? "rtl" : "ltr";
-
-      if (newDirection !== currentDirection) {
-        dispatch(setLanguage(selectedLanguage));
-        toggleModal();
-      }
-
       dispatch(setLanguage(selectedLanguage));
       updateIdentify();
     }
-  }, [dispatch, selectedLanguage, toggleModal]);
+  }, [dispatch, selectedLanguage]);
 
-  const handleLanguageSelectOnChange = useCallback((l: Locale) => {
-    setSelectedLanguage(l);
+  const handleLanguageSelectOnChange = useCallback(
+    (l: Locale) => {
```

Comment:
Can you avoid one letter variable name, please? Here it's not really problematic but keep it as a good practice for readability. for example if you have a multi-line function and then you read:
`const newDirection = i18next.dir(l);`
It can be hard to understand what is `l`

Reply:
Yes of course :) 

**_Leave the code cleaner than when you first came onto it_**